src/test/test_hs_service.c

```diff
@@ -2153,15 +2153,15 @@ test_export_client_circuit_id(void *arg)
   export_hs_client_circuit_id(edge_conn, service->config.circuit_id_protocol);
   cp1 = buf_get_contents(conn->outbuf, &sz);
   tt_str_op(cp1, OP_EQ,
-            "PROXY TCP6 fc00:dead:beef:4dad::0:ffff ::1 65535 42\r\n");
+            "PROXY TCP6 fc00::0:0:0:0:0:ffff ::1 65535 42\r\n");
   tor_free(cp1);
 
   /* Check that GID with UINT16_MAX + 7 works. */
   or_circ->global_identifier = UINT16_MAX + 7;
 
   export_hs_client_circuit_id(edge_conn, service->config.circuit_id_protocol);
   cp1 = buf_get_contents(conn->outbuf, &sz);
-  tt_str_op(cp1, OP_EQ, "PROXY TCP6 fc00:dead:beef:4dad::1:6 ::1 6 42\r\n");
+  tt_str_op(cp1, OP_EQ, "PROXY TCP6 fc00::0:0:0:0:1:6 ::1 6 42\r\n");
 
```

Comment:
Please add tests for the 2 new options, not just the old ones.

Reply:
Fixed in `4a80f4f`.